test/V4/PeanutV4.t.sol

```diff
@@ -15,7 +15,14 @@ contract PeanutV4Test is Test {
 
     // a dummy private/public keypair to test withdrawals
     address public constant PUBKEY20 = address(0xaBC5211D86a01c2dD50797ba7B5b32e3C1167F9f);
-    bytes32 public constant PRIVKEY = 0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa;
```

Comment:
`PRIVKEY` was not used before, so I used it in testMakeDepostERC20Authorization and also added `SAMPLE_ADDRESS`, which is the address that this private key controls.